.devcontainer/Dockerfile

```diff
@@ -19,11 +18,29 @@ RUN dnf update -y && \
     autoconf \
     automake \
     curl-devel \
-    libgeoip-devel \
-    liblmdb-devel \
-    libpcre++-devel \
     libtool \
     libxml2-devel \
-    libyajl \
     pkgconf \
     wget
+
+# libzip-devel from crb repo
+RUN dnf config-manager --set-enabled crb && \
+    dnf install -y libzip-devel
+
+# enable devel repo and install devel packages
+RUN dnf config-manager --set-enabled devel && \
+    dnf install -y \
+    lmdb-devel \
+    yajl-devel \
+    pcre-devel
+
+# download and install the geoipupdate rpm 
```

Comment:
is this the maxmind db aka a bunch of ip:city/country mappings?  if so we evenutally need a solution for keeping this always up to date. for dev env it doesnt matter though.

Reply:
https://github.com/maxmind/geoipupdate/releases/tag/v6.0.0
https://github.com/maxmind/geoipupdate/tags

yeah heres where I got from